admin/client/src/components/Accordion/AccordionGroup.js

```diff
@@ -15,12 +15,10 @@ class AccordionGroup extends SilverStripeComponent {
       'aria-labelledby': headerID,
     };
     return (
-      <div className="accordion-group">
-        <h6 className="accordion-group__title" role="tab" id={headerID}>
-          <a data-toggle="collapse" href={href} aria-expanded="true" aria-controls={listID}>
-            {this.props.title}
-          </a>
-        </h6>
+      <div className="accordion">
```

Comment:
Caught up with Paul about this offline, we agreed that `AccordionGroup` and `Accordion` belong together, but `AccordionItem` should be a separate component since list item styles should be separate from their placement in an accordion. I think Paul's done some work around renaming `AccordionGroup` to `AccordionBlock` since it is more consistent with bootstrap examples on how to use the collapsible JS behaviour as a de-facto accordion.